cpp/src/slice2cs/Gen.cpp

```diff
@@ -317,26 +317,31 @@ Slice::CsVisitor::writeUnmarshalDataMember(
 void
 Slice::CsVisitor::writeInheritedOperations(const InterfaceDefPtr& p)
 {
```

Comment:
I think it would be cleaner to eliminate this helper function (called once) and just call allOperations on the interface in the calling function.

Reply:
I didn't want to make it too hard to review the changes, but you're right.
I 'inlined' this code, and actually managed to simplify a good chunk of the code around where we called this too.